ThermalTalk.Imaging/ThermalTalk.Imaging.csproj

```diff
@@ -1,6 +1,18 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+<!--  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />-->
+
+  <PropertyGroup>
+    <SemVer>2.0.0.0</SemVer>
+
+    <Title>ThermalTalk.Imaging</Title>
+    <TargetFrameworks>net451;netstandard2.0;net5.0</TargetFrameworks>
+
+    <Authors>Pyramid Technologies</Authors>
+    <Company>Pyramid Technologies Inc.</Company>
+    <Description>Imaging module for ThermalTalk SDK</Description>
+    <Copyright>$(Year) Pyramid Technologies</Copyright>
```

Comment:
year is not defined in the property group

Reply:
Or does is that an auto variable? That would be cool if we don't have to manually provide that.